roles/common/templates/etc/hosts

```diff
@@ -29,7 +29,7 @@ ff02::2 ip6-allrouters
 
 {% if 'compute' in groups %}
 {% for host in groups['compute'] if inventory_hostname in groups['compute'] %}
-{{ hostvars[host].ansible_default_ipv4.address }} {{ hostvars[host].ansible_nodename }}
+{{ hostvars[host].ansible_default_ipv4.address }} {{ hostvars[host].ansible_nodename }} {{ hostvars[host].hostname_fqdn }} 
```

Comment:
We should not need this change. There is probably some other underlying root-cause for live migration to not work. I will be debugging this further tonight and provide more inputs.